pyintegration/tests/test_table.py

```diff
@@ -297,6 +299,44 @@ def test_snapshot(self):
             snapshot = t.snapshot(source_table=self.test_table, do_init=True)
             self.assertEqual(self.test_table.size, snapshot.size)
 
+    def test_agg_all_by(self):
+        test_table = empty_table(10)
+        test_table = test_table.update(["grp_id=(int)(i/5)", "var=(int)i", "weights=(double)1.0/(i+1)"])
+
+        aggs = [group(),
+                avg(),
+                first(),
+                last(),
+                max_(),
+                median(),
+                min_(),
+                pct(0.20),
+                std(),
+                sum_(),
+                abs_sum(),
+                var(),
+                weighted_avg("var"),
+                ]
+        for agg in aggs:
+            with self.subTest(agg):
+                result_table = test_table.agg_all_by(avg(), ["grp_id"])
+                self.assertGreaterEqual(result_table.size, 1)
+
+        # column names in the Aggregation are ignored
+        aggs = [group(["aggGroup=var"]),
+                avg(["aggAvg=var"]),
+                pct(0.20, ["aggPct=var"]),
+                std(["aggStd=var"]),
+                sum_(["aggSum=var"]),
+                abs_sum(["aggAbsSum=var"]),
+                var(["aggVar=var"]),
+                weighted_avg("var", ["weights"]),
+                ]
+        for agg in aggs:
+            with self.subTest(agg):
+                result_table = test_table.agg_all_by(avg(), ["grp_id"])
+                self.assertGreaterEqual(result_table.size, 1)
```

Comment:
Test looks wrong.  Do you mean:

```suggestion
        for agg in aggs:
            with self.subTest(agg):
                result_table = test_table.agg_all_by(agg, ["grp_id"])
                self.assertGreaterEqual(result_table.size, 1)
```